data/json/recipes/food/veggy.json

```diff
@@ -0,0 +1,237 @@
+      [ [ "veggy_wild", 1 ] ]
+    ]
+  },
+  {
+    "type" : "recipe",
+    "result": "cooked_cattail_stalk",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "byproducts": "plant_fibre",
+    "skills_required": [ "survival", 1 ],
+    "time": 18000,
+    "autolearn": true,
+    "qualities": [
+      { "id":"CUT","level":1 },
+      { "id":"COOK","level":1 }
+    ],
+    "tools": [ [ [ "hotplate", 5 ], [ "char_smoker", 5 ], [ "toolset", 5 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 5 ] ],
+      [ [ "cattail_stalk", 1 ] ]
+    ]
+  },
+  {
+    "type" : "recipe",
+    "result": "cooked_pumpkin",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "time": 15000,
+    "reversible": false,
+    "autolearn": true,
+    "qualities" : [
+      { "id":"CUT","level":1 },
+      { "id":"COOK","level":2 }
+    ],
+    "tools": [ [ [ "hotplate", 5 ], [ "char_smoker", 5 ], [ "toolset", 5 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 5 ] ],
+      [ [ "pumpkin", 1 ], [ "irradiated_pumpkin", 1 ] ]
+    ]
+  },
+  {
+    "type" : "recipe",
+    "result": "pie",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": 25000,
+    "reversible": false,
+    "autolearn": true,
+    "qualities" : [ { "id":"COOK","level":2 } ],
+    "tools": [
+      [ [ "hotplate", 6 ], [ "char_smoker", 6 ], [ "toolset", 6 ], [ "fire", -1 ] ],
+      [ [ "aluminum_foil", 6 ] ]
+    ],
+    "components": [
+      [ [ "flour", 2 ] ],
+      [ ["apple", 2], ["apple_canned", 1], ["apple_vac", 1], ["apricot", 2], ["banana", 2], ["blackberries", 2], ["blueberries", 2], ["cherries", 2], ["coconut", 2], ["cranberries", 6], ["dry_fruit", 2], ["grapefruit", 2], ["grapes", 4], ["irradiated_apple", 2], ["irradiated_apricot", 2], ["irradiated_banana", 2], ["irradiated_blackberries", 2], ["irradiated_blueberries", 2], ["irradiated_cherries", 2], ["irradiated_cranberries", 6], ["irradiated_grapefruit", 2], ["irradiated_grapes", 4], ["irradiated_kiwi", 2], ["irradiated_lemon", 2], ["irradiated_mango", 2], ["irradiated_melon", 4], ["irradiated_orange", 2], ["irradiated_papaya", 2], ["irradiated_peach", 2], ["irradiated_pear", 2], ["irradiated_pineapple", 2], ["irradiated_plums", 2], ["irradiated_pomegranate", 2], ["irradiated_raspberries", 2], ["irradiated_rhubarb", 4], ["irradiated_strawberries", 2], ["irradiated_watermelon", 1], ["can_peach", 2], ["juice_pulp", 4], ["kiwi", 2], ["lemon", 2], ["mango", 2], ["melon", 4], ["orange", 2], ["papaya", 2], ["peach", 2], ["pear", 2], ["pineapple", 2], ["plums", 2], ["pomegranate", 2], ["raspberries", 2], ["rhubarb", 4], ["strawberries", 2], ["watermelon", 1] ],
```

Comment:
A corner case, but ugly no doubt. 

Arguably there's a lot of redundancy in that `recipe`. We need a `using: fruit` group or something of that nature. What groups have you identified so far?